rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -102,6 +107,9 @@ public void getProjectReport(
                     case LICENSES:
                         getLicensesReports(response, sw360User, module);
                         break;
+                    case LICENSES_RESOURCE_BUNDLE:
```

Comment:
Please add new possible value to `allowableValues` in the OpenAPI doc for the function.

Reply:
@keerthi-bl please update the documentation.